examples/notebooks/tour-of-beam/getting-started.ipynb

```diff
@@ -0,0 +1,1021 @@
vtGzyO2D9a/qr7yxD/x3tnJGZDfNCFjip8ZGwalt12D6xbt05bz0MDv7RkyEbw+LFhXVTNdGNT1ZjFfgLq9x1dPTuGWlb7f8PlSi32yx2+z9asP6v/4EFNZU9LB4/n53ymrSVjU9tVWGlhAU+DR8d9Gx88MP6QnoWby64hgAACthEgeLRNqdhRBBBAAAG7Cly7dk37h/3gtFGWD9pWH7/jxP60pt7n/f7p1+dO291/4aHP23UnOPTlOR/m6U+Gz5w506k9rLC+wK6dO7VjM2ZKnC2DuXFVE6S+qd4Qfmf1bglba//LyDpmOLb/GrEwWltLxqZhd7D0A0Zjc1VEX8OAyzHDriu+EjwaB48rI/ppx6aqMYv9BNQxVRc8/jT3U0uHbK+/Ha/FfnVomM/73Xh9j/O2mxt83m6gw9st47/S1pKx6VxO1lhbwJ3gMTs7m/s2WruM7B0CCAShAMFjEBaVJiGAAAIIWEugpaVF+4d94pA0ywdtxRsuazH7Tj/h9b7/a0SVNDW3Om135o6rXm/TlzDRk9d+kV2qreXmzZud2sMK6wsYfSggfniC7YLHkQfGyutG50sjqyo0tzbLotNLbdem9sGiq+9VWKw7Gb59u/7eW9bvnaG9h0Zjc0psBMFjh65h9UutTozVz0a+fFn/+0WH5vFfiwkYfShgx8Q+lg7aXm6P0UrWn1no8343P3a+b3nLi3s+bzfQweOMwnDt++bVq8b3h9YishKBLhYwCx4XLlzIfRu7uD78eAQQCF0BgsfQrT0tRwABBBDoRAGjT4h/kLfO0mHbFxOPSatzRiiTtlzxer/TFp/RysfM/8HrbXoSHvry3H7pE7UnadTldFnsJ9DU1KStp7pXoKugy0qPD9s3Ul42vNQWoK6pTiYfnWar9nhjG1+UoK3llStXtC6stLaA0djMTEq0ZPB4OytGHs6Z6Jd/f6xerC3Os63rtNu/Gd/PkiaOmaeqZroPBdTW1mrbyUprCxh9KKBk2GBrB20r3pfWeufLkDc/uerzfrfWOV/Fo+n+KZ+3G+jgUd2bUzc21fGXBQE7CXQMHqdNmyaHDx/mvo12KiL7igACQSlA8BiUZaVRCCCAAAJWEzD6hHjP7IWWD9t0l0U9Uf2H1/u9/rvfnMpT39gs7ww/6PU2fQkTPXltVGqh9iTNs2fPnNrECnsIGH0oIGKRPe7zmLY9S17U12ix1Xp1z0dvgjy7vSYpPUk7NhsaGrQ2rLS+gNHY/HZAD0sHbY7Azduvd3LitcW5V1xou3YfHdhTOy7V/bVY7CnQ2NiorWleerzlg7aGav3VKWo39fd631/uiNUWsu7kNK+3GejAUW3/0rxPtHWcOnWqtj2sRMDKAip4HDVqlKirXNy/f9/Ku8q+IYAAAiElQPAYUuWmsQgggAACXSVQXV2t/QO/f/oEy4dt6hKoHRc1CzJs9kmP9/2z4iOiQsaOy4aTv3m8LU8CQ3889x8Fu7Q1HD58eMfm8H8bCRh9KCB6ZqzlA7uwdVFS/dh5fCr+v+r+kvzdQy3fBn8EnBHl+vs7cgLVRgNRs6tGY3ND+Ne2C+A8CSGDKXhcE/GN9n1z6dKlmoqzyi4CRh8KuDq/m6XDtpe7ErXE6h6N3oZ9zY8uOG+zpVFq1nzu9Ta93RdPXrdzYh/t2Ny4caNze1iDgMUFuDywxQvE7iGAQMgKEDyGbOlpOAIIIIBAZwq8fv1a+wd+TEqu5QM3o8utHr/i+azHdSd+dWJvbmmVPpOPW96hW26Ftobl5eVObWKFfQR+/vlnbV3jxsRbPrTbdGmLFlrd03Fs1XjL778/KufkLQAAIABJREFUQke1jejpsdoacgJV2z1ss9JobM6NHkzw+M3HtjCYFROmHZtVVVW26YfsqLPAzp07tXU9MKWXpcM2Fcy1vLjr3CARebUvw+N9f31Y/8Gzpl+Pe7wtT0JDfzy3dPhAbQ0vXNAEqVoxViKAAAIIIIAAAuYCBI/mPjyKAAIIIICA3wSMPiH+ft5Gy4duO37UX7amdN91t/d9ROVPWsu95x+6vQ1/zFz0dhtfZs3XnqTZu3evtl2stIfAy5cvtXVNzEm0dHCnZjO2tLZokVedW2PpffdX4OjYTtzoeG0NufeqtnvYZqXR2ByWEGuL0M2TWY7tnxtMMx4LE+K0Y/PGjRu26YfsqLPApUuXtHVdMnKA5QO3V1WFzg0SkZba36V280C39792w1fSUvtQu61Xez0PMf0RJnqyjeGZ+g/scO9VbUlZiQACCCCAAAJeCBA8eoHGSxBAAAEEEPBGYOvWrdoTNT2yyy0fvH067oi8eN3o1Gx1ydX5e6/J34buN23D2PWXpKHJOSRRrx846zvT13obFPr7df3SJ2rrd+XKFScXVthLwOhDAeFLrHufx+9+PalF/vX5b1J8aKLf/8VvTrZsmKlC4uTkZKd/L1680Bqx0j4CRmPz5IDPgzZ8DJbg8bsBPZzGpBqnWVlZ0tTUZJ9OyJ46CdTU1GhrOzo7xu3gzpOQzN/Pbbx9yKlNakVr/Qt5uSfFZRteHciR1vq/tNto+GWby9f7uz2ebu/a/I+19ePy5NqSshIBBBBAAAEEvBQgePQSjpchgAACCCDgqcD58+e1f+gPSC+2RfA2dWu1YZNP33gqaYvPOAWQ0SU/SNVPjwxf58mMSX8HiZ5s7x8FuyU2JcepfmlpafLq1SvD9vGAPQQ2b97sVFt1gjxmapwlw7acXQWGsx0DJT7h8GRLWkSWxmhrxwnUQPWEzt2u0dhcHdGX4NHil1tVNdJ9IECFySz2F5g+fbq2vufmfGb54K2m8gtprfvToAit0vTbt/Jy9xB5sezd/7Vl5UfycneyNFxeqyJK7WvVDMiaik/+95pl71jy++0TuL+jtoCsRAABBBBAAAG/ChA8+pWTjSGAAAIIIGAs8Pz5c+1JGnVi7oO89ZYPH/8+9IAcvGgcIqqWq1mRV++/kMu//SXPahuMMURkyw93Ld9mRz
```

Comment:
Got it, I'll remove the "Note" and make sure all of them are capitalized.

Unfortunately adding an extra space doesn't do anything since Markdown/HTML combine one or more spaces and render them as only one. But I can always add the ugly `&nbsp;` to give them an extra space, it does help on the visualization so I'll add them.